src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -771,16 +815,32 @@ namespace FanCoilUnits {
                     for (NodeNum = 1; NodeNum <= state.dataZoneEquip->ZoneEquipConfig(CtrlZone).NumExhaustNodes; ++NodeNum) {
                         if (FanCoil(FanCoilNum).AirInNode == state.dataZoneEquip->ZoneEquipConfig(CtrlZone).ExhaustNode(NodeNum)) {
                             state.dataFanCoilUnits->ZoneExNodeNotFound = false;
+                            break;
                         }
                     }
+                    if (!state.dataFanCoilUnits->ZoneExNodeNotFound) break;
                 }
                 if (state.dataFanCoilUnits->ZoneExNodeNotFound) {
-                    ShowSevereError(state,
-                                    CurrentModuleObject + " = \"" + FanCoil(FanCoilNum).Name +
-                                        "\". Fan coil unit air inlet node name must be the same as a zone exhaust node name.");
-                    ShowContinueError(state, "..Zone exhaust node name is specified in ZoneHVAC:EquipmentConnections object.");
-                    ShowContinueError(state, "..Fan coil unit air inlet node name = " + state.dataLoopNodes->NodeID(FanCoil(FanCoilNum).AirInNode));
-                    ErrorsFound = true;
+                    bool InletNodeFound = false;
+                    for (CtrlZone = 1; CtrlZone <= state.dataGlobal->NumOfZones; ++CtrlZone) {
+                        if (!state.dataZoneEquip->ZoneEquipConfig(CtrlZone).IsControlled) continue;
+                        InletNodeFound = isReturnPlenumInducedNode(state,
+                                                                   FanCoil(FanCoilNum).AirInNode,
+                                                                   state.dataZoneEquip->ZoneEquipConfig(CtrlZone).NumReturnNodes,
+                                                                   state.dataZoneEquip->ZoneEquipConfig(CtrlZone).ReturnNode);
```

Comment:
Looking at this more, `isReturnPlenumInducedNode` is not doing much. How about 
`ZonePlenum::ValidateInducedNode( state, InletNodeNum,  state.dataZoneEquip->ZoneEquipConfig(CtrlZone).NumReturnNodes, state.dataZoneEquip->ZoneEquipConfig(CtrlZone).ReturnNode)` right here. With the list of return nodes passed in, `ValidateInducedNode` can search the plenum induced air nodes first, then look for matching return node to plenum inlet node once the correct plenum is located.